docs/general.md

```diff
@@ -0,0 +1,43 @@
+# General
+For whom is this documentation intended? x y and z so we will cover ... for a reason.
```

Comment:
Would you please add an admonition to present this as draft content?

It's ok to include this in the PR but please use an admonition to clearly signpost is as such.

They look like

```
:::{note}
This text is **standard** _Markdown
:::

```

More below

https://myst-parser.readthedocs.io/en/latest/syntax/optional.html?highlight=admonition#admonition-directives